src/layer/marker.ts

```diff
@@ -204,8 +204,8 @@ export class Marker extends Layer<DivIconMarker> {
     ) {
         super();
 
-        const marker = this.map.plugin.getIconForType(type);
-        if (!marker) {
+        const markerIcon = this.map.markerIcons.get(type) ?? this.map.markerIcons.get('default');
+        if (!markerIcon) {
```

Comment:
Note that this code will probably never run? But it wasn't running before either, since `plugin.getIconForType` also returned the default icon if none was found for the given type.